book/SUMMARY.md

```diff
@@ -148,19 +153,19 @@
 
 - [Introduction](/quick-brick/intro.md)
 - [Apple TV](/quick-brick/tvOS/intro.md)
-	- [Plugins](/quick-brick/tvOS/Plugins/intro.md)
-		- [Analytics](/quick-brick/tvOS/Plugins/Analytics/intro.md)
-            - [Plugin creation guide](/quick-brick/tvOS/Plugins/Analytics/GeneralAnalyticsHowCreate.md)
-            - [Google Analytics](/quick-brick/tvOS/Plugins/Analytics/GoogleAnalytics/GoogleAnalytics.md)
-        - [Player](quick-brick/tvOS/Plugins/Video/intro.md)
-            - [Plugin creation guide](quick-brick/tvOS/Plugins/Video/VideoPluginsHowCreate.md)
-            - [Default Player](quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
-        - [Player Dependent](quick-brick/tvOS/Plugins/PlayerDependents/intro.md)
-            - [Plugin creation guide](quick-brick/tvOS/Plugins/PlayerDependents/DependentPluginsHowCreate.md)
-            - [GoogleIMA](quick-brick/tvOS/Plugins/PlayerDependents/VideoAds/GoogleIMA/GoogleIMA.md)
-	- [Environment](/quick-brick/tvOS/Environment/intro.md)
-    	- [Apple TV Debug](/quick-brick/tvOS/Environment/DebugBuildIntegration.md)
-        - [Apple TV Release](quick-brick/tvOS/Environment/ReleaseBuildIntegration.md)
+  - [Plugins](/quick-brick/tvOS/Plugins/intro.md)
```

Comment:
quick-brick plugins should be inside the plugins section. 
for example qb login should be inside plugins/login with ios and android